test/common/common/hex_fuzz_test.cc

```diff
@@ -0,0 +1,14 @@
+#include "common/common/hex.h"
+
+#include "test/fuzz/fuzz_runner.h"
+
+namespace Envoy {
+namespace Fuzz {
+
+DEFINE_FUZZER(const uint8_t* buf, size_t len) {
```

Comment:
Would you be able to also include coverage over `uint64ToHex` and `uint32ToHex`? (An easy way to get ints from raw buffers is to use `FuzzedDataProvider::ConsumeIntegral<uint32>`) 

See https://github.com/google/fuzzing/blob/master/docs/split-inputs.md#fuzzed-data-provider
https://github.com/envoyproxy/envoy/blob/3662462563f4b98fde7935215e551582f7e4ea90/test/common/compressor/compressor_fuzz_test.cc

Reply:
Thanks for the information.
Tests added.